README.md

```diff
@@ -0,0 +1,56 @@
+# Astro Starter Kit: Basics
```

Comment:
At some point we should probably customize this for the site, looks okay for now tho!